src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -156,6 +155,7 @@ class Diagram extends Component<Props> {
       getCurrentFlow: () => this.getPropsProperty('currentFlow'),
       updateFlowNode: this.updateNodeAndRefresh.bind(this),
       switchFlowNode: this.switchFlowNode.bind(this),
+      getDefaultLanguage: () => this.getPropsProperty('defaultLanguage'),
       getCurrentLang: () => this.getStateProperty('currentLang'),
```

Comment:
What do you think of merging both of them, call it `getLanguage` and pass around an object {current, default} instead of passing two new variables everywhere ?